compute/resource_job.go

```diff
@@ -126,6 +126,14 @@ var jobSchema = common.StructToSchema(JobSettings{},
 				return false
 			}
 		}
+		s["email_notifications"].DiffSuppressFunc = func(k, old, new string, d *schema.ResourceData) bool {
+			log.Printf("[INFO] k='%v', old='%v', new='%v'", k, old, new)
+			if old == "1" && new == "0" {
```

Comment:
`no_alert_for_skipped_runs` is an integer?

Reply:
No. This field is a list, so it receives length of that list, and diff on individual fields isn’t called (maybe it’s called if the predefined value changes, but it should be handled by default parameters).